sdv/constraints/tabular.py

```diff
@@ -852,6 +829,10 @@ def __init__(self, low_column_name, middle_column_name, high_column_name,
         self.low_column_name = low_column_name
         self.middle_column_name = middle_column_name
         self.high_column_name = high_column_name
+        self.diff_column_name_1 = f'{self.low_column_name}#{self.middle_column_name}'
+        self.diff_column_name_2 = f'{self.middle_column_name}#{self.high_column_name}'
```

Comment:
minor: can we call them `low_diff_column_name` and `high_diff_column_name`?

Reply:
Yes done in [18bdaf0](https://github.com/sdv-dev/SDV/pull/1404/commits/18bdaf09a65211edb2e8c54ccffd057a72fa68d4)